buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -391,6 +391,17 @@ class ClusterFormationTasks {
         return configureExecTask(name, project, setup, node, args)
     }
 
+    /** Surround strings that contains a comma with double quotes **/
+    private static String escapeComma(Object o) {
+        if (o instanceof String) {
+            String s = (String)o
```

Comment:
This will cause args to be forced into a String at configuration time. in the case of plugins, it will cause them to resolve early, but it needs to be lazy resolution. In order to do this still lazily, we would need to wrap each arg with a wrapper that, on call to toString(), does the escaping if necessary.